db/migrate/20220806202153_remove_is_tracker_from_domains.rb

```diff
@@ -0,0 +1,7 @@
+class RemoveIsTrackerFromDomains < ActiveRecord::Migration[7.0]
+  def change
+    if ActiveRecord::Base.connection.column_exists?(:domains, :is_tracker)
+      remove_column :domains, :is_tracker, :boolean
```

Comment:
Move this `remove_column` up into the prior migration, there isn't a benefit to having two migrations.

Reply:
OK